code/playerman/player.h

```diff
@@ -52,6 +52,9 @@
 #define PLAYER_FLAGS_KILLED_SELF_UNKNOWN			(1<<16)		// player died by his own hand
 #define PLAYER_FLAGS_KILLED_SELF_MISSILES			(1<<17)		// player died by his own missile
 #define PLAYER_FLAGS_KILLED_SELF_SHOCKWAVE		(1<<18)		// player died by his own shockwave
+#define PLAYER_FLAGS_PLR_VER_CONTROLS		(1<<19)		// loaded PLR file's plr_ver is a pre-controls5 version
+#define PLAYER_FLAGS_PLR_VER_LOWER			(1<<20)		// loaded PLR file's plr_ver is lower than PLR_VERSION
+#define PLAYER_FLAGS_PLR_VER_HIGHER			(1<<21)		// loaded PLR file's plr_ver is higher than PLR_VERSION
```

Comment:
I'd rename these.  `PLAYER_FLAGS_PLR_VER_CONTROLS` should be `PLAYER_FLAGS_PLR_VER_OLD_CONTROLS` or maybe `PLAYER_FLAGS_PLR_VER_PRE_CONTROLS5`.  Something a little more descriptive.  The other two are better but I would prefer `PLAYER_FLAGS_PLR_VER_IS_LOWER` and `PLAYER_FLAGS_PLR_VER_IS_HIGHER`